app/models/course/group_user.rb

```diff
@@ -4,9 +4,9 @@ class Course::GroupUser < ActiveRecord::Base
 
   enum role: { normal: 0, manager: 1 }
 
-  validate :user_and_group_in_same_course
+  validate :course_user_and_group_in_same_course
 
-  belongs_to :user, inverse_of: :course_group_users
+  belongs_to :course_user, inverse_of: :group_users
   belongs_to :course_group, class_name: Course::Group.name, inverse_of: :group_users
```

Comment:
but now that we are doing this on the course_user model, that statement no longer holds?